Makefile

```diff
@@ -249,6 +249,13 @@ start_local:
 start_local_debug:
 	operator-sdk run --local --watch-namespace $(NAMESPACE) --enable-delve 2>&1 | grep --color=always -E '"msg":"[^"]*"|$$'
 
+### test_e2e: runs e2e test on the cluster set in context. Includes deploying devworkspace-controller, run test workspace, uninstall devworkspace-controller
+test_e2e: _print_vars _set_ctx _update_yamls update_devworkspace_crds _do_e2e_test _reset_yamls _reset_ctx
+
+_do_e2e_test:
```

Comment:
I think it might be cleaner to have all the hidden rules (`_*`) grouped together

Reply:
I think putting such _do methods could help understand main target logic but agree that according to the current style - it should be moved above to there _*